diff --git a/Engine/Source/Programs/UnrealBuildTool/Platform/Linux/UEBuildLinux.cs b/Engine/Source/Programs/UnrealBuildTool/Platform/Linux/UEBuildLinux.cs
index 0bc848fb4ed..9f7332ed8fc 100644
--- a/Engine/Source/Programs/UnrealBuildTool/Platform/Linux/UEBuildLinux.cs
+++ b/Engine/Source/Programs/UnrealBuildTool/Platform/Linux/UEBuildLinux.cs
@@ -654,7 +654,7 @@ namespace UnrealBuildTool
 		/// <summary>
 		/// Force using system compiler and error out if not possible
 		/// </summary>
-		private int bForceUseSystemCompiler = -1;
+		private int bForceUseSystemCompiler = 1;
 
 		/// <summary>
 		/// Whether to compile with the verbose flag
